src/scripts/release.zig

```diff
@@ -615,6 +616,52 @@ fn publish_node(shell: *Shell, info: VersionInfo) !void {
     });
 }
 
+fn publish_docker(shell: *Shell, info: VersionInfo) !void {
```

Comment:
I know you said your reasoning for not having a `build_docker` step - but there are still things that can go wrong in the `Dockerfile` build even though we aren't really doing much besides copying a binary...